nextcord/http.py

```diff
@@ -476,13 +503,10 @@ def send_multipart_helper(
         stickers: Optional[List[sticker.StickerItem]] = None,
         components: Optional[List[components.Component]] = None,
         attachments: Optional[List[dict]] = None
-    ) -> Response[message.Message]:
+    ):
```

Comment:
Typehint response type, why was it removed?

Reply:
This should be list[dict], github shows it like I removed the typehint of send_multipart_helper, but it's still there. The thing where it shows the change is now a complete new function "get_multipart_form" to handle stuff with multiparts easier

Reply:
resolved in `eec0812`, typehinted with list[dict]